### PR TITLE
Statvar Readme files for Brazil Import

### PR DESCRIPTION
**Pull request by HarishC727**

### CODE REVIEW

File: statvar_imports/brazil_visdata/FoodBasketDistribution/common_metadata.csv [hunk 1]
```diff
@@ -0,0 +1,17 @@
+parameter,value
+url,
+description,Acquisition of Basic Baskets for families for Specific Population
+#place_type,
+#places_within,
+start_date,
```
Comment (ajaits) [inline statvar_imports/brazil_visdata/FoodBasketDistribution/common_metadata.csv:6]: Let's add a reference to the places_csv
```
places_resolved_csv,../Brazil_Places_Resolved.csv
```

**Review by ajaits — APPROVED** [review]

File: statvar_imports/brazil_visdata/FoodBasketDistribution/common_metadata.csv (hunk 1 above)
Comment (HarishC727) [reply statvar_imports/brazil_visdata/FoodBasketDistribution/common_metadata.csv:6]: Can we keep it at both places - Metadata and the in the command line?
places_resolved_csv does not get recognised when placed inside the metadata but it works fine when passed as a flag in the command.

**Pull request merged**